content/en/docs/tasks/administer-cluster/topology-manager.md

```diff
@@ -223,12 +223,11 @@ You can toggle groups of options on and off based upon their maturity level usin
 
 You will still have to enable each option using the `TopologyManagerPolicyOptions` kubelet option.
 
-### `prefer-closest-numa-nodes` (beta) {#policy-option-prefer-closest-numa-nodes}
+### `prefer-closest-numa-nodes` (stable) {#policy-option-prefer-closest-numa-nodes}
```

Comment:
Adding "(stable)" is optional.
For most, if not all, features that are stable, we don't explicitly mention it.

Reply:
👍 will change that